src/components/views/rooms/RoomTile.tsx

```diff
@@ -470,7 +481,11 @@ export default class RoomTile extends React.PureComponent<IProps, IState> {
                         label={lowPriorityLabel}
                         iconClassName="mx_RoomTile_iconArrowDown"
                     />
-
+                    <IconizedContextMenuOption
+                        onClick={this.onInviteClick}
+                        label={_t("Invite to this room")}
+                        iconClassName="mx_RoomTile_iconInvite"
+                    />
```

Comment:
this should probably check whether the user has permission to complete that action

Reply:
Oh okay will do that

Reply:
Is there any code formatter that can be used for this project. I am used to them and can't find anything which works well with this project's code.

Reply:
@jaiwanth-v, I use [this](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) extension for VS Code but it's not perfect. For other questions like this, I would recommend the [#element-dev](https://matrix.to/#/#element-dev:matrix.org?via=matrix.org&via=vector.modular.im&via=mozilla.org) room

Reply:
Thanks👍